hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -198,8 +199,10 @@ MDMParser::MDMParser(void)
     inst = this;
     memset(&_dev, 0, sizeof(_dev));
     memset(&_net, 0, sizeof(_net));
-    _net.lac = 0xFFFF;
-    _net.ci = 0xFFFFFFFF;
+    _net.cgi.mobile_country_code = 0;
```

Comment:
zeroing here should be unnecessary, as we've just called `memset`

Reply:
Agreed, but we also set `false` to a bunch of variables, that's what threw me off.